test/test-parse-expression.cpp

```diff
@@ -1442,6 +1442,21 @@ TEST_F(test_parse_expression, malformed_object_literal) {
                                 error_invalid_lone_literal_in_object_literal,
                                 where, offsets_matcher(p.locator, 1, 4))));
   }
+
+  {
+    test_parser p(u8"{`key`: 1};"_sv);
+    expression_ptr ast = p.parse_expression();
+    EXPECT_EQ(summarize(ast), "object(literal, ?)");
```

Comment:
This looks unexpected to me. If we see a template literal followed by a `:`, I think we should recover by pretending that the template literal was a key. Therefore, this assertion should say something like `object(literal, literal)`.